code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -825,6 +825,39 @@
 	L.regenerate_icons()
 	qdel(src)
 
+/obj/item/slimepotion/slime/renaming
+	name = "renaming potion"
+	desc = "A potion that allows a self-aware being to change what name it subconciously presents to the world."
```

Comment:
But you can use it on other mobs?

Reply:
Yes?

Reply:
Self-Aware is confusing considering L926 disallows sentienced name-changing